users/service.go

```diff
@@ -64,10 +44,10 @@ func (svc service) RegisterClient(ctx context.Context, token string, cli mgclien
 	if !svc.selfRegister {
 		userID, err := svc.Identify(ctx, token)
 		if err != nil {
-			return mgclients.Client{}, err
+			return mgclients.Client{}, errors.Wrap(svcerr.ErrAuthorization, err)
```

Comment:
In Line 45 `svc.Identify(ctx, token)` already wraps `svcerr.ErrAuthorization`, so need to wrap it again here 
If the functions call is goes out the package then wrapping is required, function call is within the package, the called function should wrap right  errors